core/src/main/scala/kafka/log/Log.scala

```diff
@@ -531,65 +531,94 @@ class Log(@volatile var dir: File,
    * by this method.
    * @return Set of .swap files that are valid to be swapped in as segment files
```

Comment:
I think the @return doc of this function is outdated
It's not returning files anymore, I think it's now returning file offsets

Reply:
Yes, you're right. I forgot to update it. And now it's done